### ISSUE TITLE
remove any tokens that exist as diamond facets and remove old tokens

### ISSUE THREAD

**Issue by zgorizzo69** [pavlovcik/uad-ui#593]
Problem is there are now two set of contracts in parallel inside folders : dollar and diamond also there is one diamond deployment and one more advanced with forge script that deploy dollar and all the other contracts 
we should also make sure that no tokens exist as a diamond facet and that all tokens uses the diamond as a manager (for access control with separated mint and burn rights)

- [x] remove any tokens that exist as diamond facets
- [x] have them use a separate manager
- [x]  remove the  contracts that exists as a diamond facet from the dollar directory

**Comment by 0x4007** [comment]
I put this as urgent because I understand that we can't get the UI stable til we get this fixed.

**Comment by zgorizzo69** [comment]
So I removed all the tokens and put them inside an old folder 
I will try to keep some old test for what we have now in production 
I have updated the remaining token to connect to the diamond
but now I have to adapt all the test for these tokens to use the diamond 
and after that I will adapt the deploy script 
I will probably leave some test refactoring for another PR cos this one is already huge

**Comment by zgorizzo69** [comment]
> Do you have any updates? @zgorizzo69

Waiting for fam to review pr #600

**Comment by 0x4007** [comment]
@zgorizzo69 just try and resolve the identifiers issue to merge and close before the bot rugs you this week lol

**Comment by 0x4007** [comment]
@0xcodercrane no payment permit was issued by the bot here. Can you throw an error or provide some type of feedback if it had a problem? Let me know what you think happened here. Thanks!

**Comment by cuddle-dev** [comment]
> @0xcodercrane no payment permit was issued by the bot here. Can you throw an error or provide some type of feedback if it had a problem? Let me know what you think happened here. Thanks!

because `autoPayMode` is false for the repository. we can enable is in .github/ubiquibot-config.yml

**Comment by 0x4007** [comment]
Oh interesting that you made a repository wide config. My vision was to have it issue specific. 

We need some adjustments: 

1. By default it should be enabled. However, once the multisig/permit idea is implemented (@rndquu is researching now) the config property should be representing the cutoff price for autopay to require approval via multisig. e.g. `autoPayThreshold: 100` to automatically pay bounties less than $100
2. We should have it configurable on the issue/bounty level because this feature is really useful to leave on EXCEPT for changes to CI. With those, its easier to test if we just merge the PR and test and then manually do the payout.

**Comment by 0x4007** [comment]
@zgorizzo69 I will manually pay this out. What is your address? Thanks

**Comment by zgorizzo69** [comment]
> @zgorizzo69 I will manually pay this out. What is your address? Thanks

0x10693e86f2e7151B3010469E33b6C1C2dA8887d6

**Comment by 0x4007** [comment]
https://etherscan.io/tx/0x74ff46854f750cf58aa6d3ab671ee70d57cb7aefd76d34f3e80231a842f9685d
